### DISCUSSION TITLE
【灌水】听说原神是最近新艾利都很流行的一款游戏，想看看绳网上对原神这款游戏的评价怎么样，再决定要不要玩

### DISCUSSION THREAD

**Discussion by Genshinimpactking** [share121/inter-knot#1507] [其他]
wuwa

**Comment by uyfh352** [comment]
聽說同一間公司出的《崩壞：星穹鐵道》也不錯，最近打算入手玩一下

**Comment by Kamisato100** [comment]
隔壁Wuwa打过来我第一个跑

**Reply by Gameclubuser** [reply]
这里不是他主场，打过来也没用

**Comment by AZEROMCN** [comment]
感觉原神……画质不如O
（玩梗开个玩笑，不要当真）

**Comment by youyiMC** [comment]
听说挂后台能增加前台游戏帧率（doge）

**Comment by qinYun256** [comment]
总之过完这个版本我就准备退坑了，9月初的时候正好有一款叫做空月之歌的新戏公测；再见了原神，希望你能够喜欢你这5年来的戏份。
向我们迎面走来的是《原神：空月之歌》！！！![1753180909485a9f0a4ad2adf10935e3b131ac991e37b76130d01c450a5d3f0b0f5d100c90696.0.jpg](https://github.com/user-attachments/assets/c38893b4-610b-4211-a0b5-879b05828672)

**Comment by BSJ3456A** [comment]
玩过 很好吃（歪比巴卜）

**Comment by hot-ice-seele** [comment]
我在外环的哥们玩过这个，我去帮你问问

**Comment by oldzhunotold** [comment]
卫非地的也玩，但他们更喜欢崩坏3，据说是非常温馨甜美，阳光开朗，积极向上的故事
